### DISCUSSION TITLE
Quicker Workflow: Browser Bookmark hook template trigger

### DISCUSSION THREAD

**Discussion by Thundercraft5** [coddingtonbear/obsidian-web#168] [Ideas]
Idea: Add a hook for when the user clicks on the "Bookmark" button in the browser, that could possibly trigger a given request template.

Use case: Make the workflow of saving webpages faster. 
Specifics: Listen for Bookmark Event --> Bookmark Event --> Activate Request Template (Set by user) --> To Obsidian LWS as normal
In the extension settings: "Template to trigger on tab bookmark: `<Template>`"

**Comment by coddingtonbear** [comment]
That's a neat idea!  I have to say, though, that I'm not sure I think that that would provide enough utility for me to be up for supporting and maintaining that extra complexity.  It just saves maybe two clicks (just a few seconds), and would at the same time cost me at least a couple _hours_ of time to put together while also adding permanent complexity burden that would have to be waded through for all future bugfixes, features, and refactorings.  Complexity like that is, for better or worse, the death of projects like this.  I appreciate the ideas, though!
